src/pages/documentation/mobile-core/configuration/index.md

```diff
@@ -0,0 +1,214 @@
+
+React Native
+
+<ProgrammaticUpdatesReactNative/>
+
+Flutter
+
+<ProgrammaticUpdatesFlutter/>
+
+Cordova
+
+<ProgrammaticUpdatesCordova/>
+
+Unity
+
+<ProgrammaticUpdatesUnity/>
+
+Xamarin
+
+<ProgrammaticUpdatesXamarin/>
+
+## Clearing programmatic updates to the configuration
+
+<InlineAlert variant="info" slots="text"/>
+
+This API is only available in Android and iOS (AEP 3.x).
+
+You can clear any programmatic updates made to the configuration via the `clearUpdatedConfiguration` API. This will clear programmatic updates to configuration made via the `updateConfiguration(configMap)`(Android)/ `updateConfigurationWith(configDict:)`(iOS) API. It will also clear any updates to the `MobilePrivacyStatus`(Android)/ `PrivacyStatus`(iOS)  made via `setPrivacyStatus(privacyStatus)`(Android)/ `setPrivacyStatus(_ status:)`(iOS).
+
+For implementation details, please refer to [Configuration API reference](./api-reference.md#clearUpdatedConfiguration).
+
+## Using a bundled file configuration
+
+You can include a bundled JSON configuration file in your app package to replace or complement the configuration that was downloaded by using the [Configure with App ID per environment](./#configure-with-app-id-per-environment) approach. FIX LINK
+
+To download the JSON configuration file, use the following URL:
+
+`https://assets.adobedtm.com/PASTE-ENVIRONMENT-ID.json`
+
+* In iOS, the `ADBMobileConfig.json` file can be placed anywhere that it is accessible in your bundle.
+* In Android, the `ADBMobileConfig.json` file must be placed in the assets folder.
+
+You can also load a different `ADBMobileConfig.json` file by using the `ConfigureWithFileInPath` method. The Adobe Experience Platform SDKs will attempt to load the file from the given path and parse its JSON contents. Previous programmatic configuration changes that were set by using the `UpdateConfiguration` method are applied on the bundled file's configuration before setting the new configuration to the Adobe Experience Platform SDKs. If a file-read error or JSON parsing error occurs, no configuration changes are made.
+
+To pass in a bundled path and file name:
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="4"/>
+
+Android
+
+<BundledFileAndroid/>
+
+iOS (AEP 3.x)
+
+<BundledFileIosAEP/>
+
+iOS (ACP 2.x)
+
+<BundledFileIosACP/>
+
+Xamarin
+
+<BundledFileXamarin/>
+
+## Environment-aware configuration properties
+
+<InlineAlert variant="info" slots="text"/>
+
+This feature is only available in iOS ACPCore version 2.0.3 or later, and iOS AEPCore version 3.0.0 and above.
+
+Some extension developers might use different configuration values based on their environment, and the generated configuration might have several entries for the same property. For example, the Adobe Campaign Standard extension has different endpoints for development, staging, and production servers. Here is an example of a raw configuration that supports multiple build environments:
+
+#### JavaScript
```

Comment:
Can you remve the Javascript heading? I also see it in our current documentation and it should not be there.